lightwood/analysis/helpers/feature_importance.py

```diff
@@ -0,0 +1,59 @@
+from copy import deepcopy
+from types import SimpleNamespace
+from typing import Dict, Tuple
+
+import torch
+import numpy as np
+import pandas as pd
+
+from lightwood.analysis.base import BaseAnalysisBlock
+from lightwood.helpers.general import evaluate_accuracy
+from lightwood.analysis.nc.util import t_softmax
+
+
+class GlobalFeatureImportance(BaseAnalysisBlock):
```

Comment:
please potentially annotate what this function does - people do care about how column importances are calculated